src/DssSpell.sol

```diff
@@ -27,36 +27,16 @@ interface RwaUrnLike {
     function draw(uint256) external;
 }
 
-interface HopeLike {
-    function hope(address) external;
-}
-
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/0845cb91e15d928e636918e9b59da83f68e86a5f/governance/votes/Executive%20vote%20-%20August%2012%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO/governance/votes/TODO.md -q -O - 2>/dev/null)"
     string public constant override description =
-        "2022-08-10 MakerDAO Executive Spell | Hash: 0xe8f7b317e95c89b79044775787087895c0b4b65486e4ad21246cfac10da2ea17";
+        "2022-08-24 MakerDAO Executive Spell | Hash: TODO";
 
-    uint256 public constant MILLION            = 10 **  6;
-    uint256 public constant BILLION            = 10 **  9;
-    uint256 public constant WAD                = 10 ** 18;
+    uint256 public constant WAD     = 10 ** 18;
```

Comment:
Should probably remove the line after this and then fix the spacing so the `=` line up (or just remove the extra spacing between `WAD` and `=` here

Reply:
Done